package.json

```diff
@@ -23,39 +23,48 @@
     "@types/react": "16.9.17",
     "@types/react-color": "3.0.1",
     "@types/react-dom": "16.9.4",
+    "eslint-plugin-json-format": "2.0.1",
     "husky": "4.0.10",
     "lint-staged": "10.0.0",
     "node-sass": "4.13.1",
     "prettier": "1.19.1",
-    "rewire": "^4.0.1",
+    "rewire": "4.0.1",
     "typescript": "3.7.5"
   },
   "eslintConfig": {
-    "extends": "react-app"
+    "extends": "react-app",
+    "plugins": [
+      "json-format"
+    ],
+    "settings": {
+      "json/sort-package-json": false
+    },
+    "rules": {
+      "no-console": [
+        "warn",
```

Comment:
```suggestion
        "error",
```

Reply:
Let's keep it as `warn` so that your editor doesn't make you think you mistyped the syntax. We prevent committing with warnings, anyway.